### PR TITLE
[22] Frontend - As a visitor, I can login to my account

### LINKED ISSUE #22: [Frontend] As a visitor, I can login to my account
As a visitor, I can log in to my account when I access `/sign_in`

## Acceptance criteria

1. Access `/sign_in`, display a form with these required fields:
- Login title
- Email
- Password
- Login button

2. Use HTML validation on the form

### PR DESCRIPTION
**Pull request by byhbt**
https://github.com/byhbt/kwtool/projects/1#card-57977870

## What happened

Implement the **UI** for the login feature.
 
## Insight

n/a
 
## Proof Of Work

Screen Shot 2021-04-01 at 10 59 37 AM

### CODE REVIEW

File: lib/kwtool_web/router.ex [hunk 1]
```diff
@@ -21,6 +21,7 @@ defmodule KwtoolWeb.Router do
 
     get "/", PageController, :index
     get "/sign_up", AuthController, :show
+    get "/sign_in", AuthController, :index
```
Comment (olivierobert) [inline lib/kwtool_web/router.ex]: Technically, sign in should be `:new` as we create a new session 💭

Comment (byhbt) [reply lib/kwtool_web/router.ex]: yes, it seems to make more sense. Updated!

File: test/kwtool_web/controllers/auth_controller_test.exs [hunk 2]
```diff
@@ -7,4 +7,11 @@ defmodule KwtoolWeb.AuthControllerTest do
       assert html_response(conn, 200) =~ "Keyword Research Tool"
     end
   end
+
+  describe "sign_in/2" do
+    test "returns 200 status", %{conn: conn} do
+      conn = get(conn, "/sign_in")
```
Comment (hanam1ni) [inline test/kwtool_web/controllers/auth_controller_test.exs:13]: Let's add space after this line to separate the section

**Review by hanam1ni — APPROVED** [review]
Small comment about styling, approved the PR

File: lib/kwtool_web/templates/auth/sign_in.html.eex [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<main class="form-authentication">
+  <h1>Login</h1>
+  <form>
+    <div class="form-group">
+      <label for="email">Email</label>
+      <input type="email" id="email" class="form-control" placeholder="Email Address" required autofocus>
```
Comment (rosle) [inline lib/kwtool_web/templates/auth/sign_in.html.eex:6]: Maybe using form helper here would be nice as later we might want to show some form changeset errors, we will eventually use form_for 💡

**Review by rosle — APPROVED** [review]

File: lib/kwtool_web/templates/auth/sign_in.html.eex (hunk 3 above)
Comment (byhbt) [reply lib/kwtool_web/templates/auth/sign_in.html.eex:6]: yes, thanks for the suggestion. I will use the form helper next time in the frontend tickets.

**Pull request merged**